weldx/transformations/cs_manager.py

```diff
@@ -610,6 +610,35 @@ def assign_data(
 
         self._graph.nodes[reference_system]["data"][data_name] = data
 
+    def delete_data(self, data_name: str):
+        """Remove the assigned data with given name.
+
+        Parameters
+        ----------
+        data_name :
+            The previously assigned data name.
+
+        Raises
+        ------
+        ValueError
+            If data_name has not been assigned yet.
+        """
+        # remove data from subsystems
+        for s in self._subsystems:
+            try:
+                s.data.remove(data_name)
+            except KeyError:
+                pass
```

Comment:
```suggestion
        for s in self._subsystems:
            s.data.discard(data_name)
```

Reply:
Didn't know about the discard method, what is the difference to remove?

Reply:
Ah discard does not raise :)

Reply:
jep, so we save some exception handling 😉